src/main/java/kostka/moviecatalog/entity/MovieDetail.java

```diff
@@ -1,5 +1,8 @@
 package kostka.moviecatalog.entity;
 
+import kostka.moviecatalog.dto.CommentDetailDto;
```

Comment:
minor: přiznám se, že se mi to teď nechce po ránu proklikávat, ale jestli je MovieDetail taky DTO, možná bych to dal do package dto, místo entity.

Reply:
máš pravdu, taky jsem to zvažoval že to přesunu :) uvidíme, co všechno se rozbije :D